### PR TITLE
bump patch version

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Bump new version so we can use the correct aerosol lookup table. I already tested that it works in ClimaAtmos.

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -14,6 +17,13 @@ memory at initialization). Now, the variable is correctly accumulated over for
 every g-point. Note, however, that only the value at the surface (`[1, :]`) is
 updated. PR [#550](https://github.com/CliMA/RRTMGP.jl/pull/550).
 
+#### Fix aerosol lookup table
+
+Use shortwave aerosol lookup table temporarily added directly to the repository.
```
Comment (Sbozzolo) [inline NEWS.md:22]: It took me a couple of minutes to understand this sentence. 

I would rephrase the paragraph to be more descriptive.
```suggestion
Starting with this release, RRTGMP.jl will use an aerosol look up table that is internally stored, as opposed
to downloading it from the `rrtgmp-data` repository. The reason for this change is that the data distributed 
with `rrtgmp-data` contains an an error in the array ordering for the aerosol optics lookup table for sea-salt (‘aero_salt_tbl’). This error was fixed in the internal table. `RRTGMP.jl` will revert to using `rrtgmp-data` 
once the repository updates their tables. PR [#548](https://github.com/CliMA/RRTMGP.jl/pull/548/).
```

**Review by Sbozzolo — APPROVED** [review]

**Comment by sriharshakandala** [comment]
Looks good to me!
